docs/pre-consumption/README.md

```diff
@@ -30,7 +30,7 @@ paperless-ngx/
 ```
 
 The pre-consumption "wrapper" just contains some helper variables and calls to your actual pre-consumption scripts.
-
+<a name="pre-consumption-wrapper" />
```

Comment:
`a` isn't a self-closing HTML element. Would you please close the link tag properly? All subsequent text is formatted as link otherwise.

Also please add a newline before and after the anchor.